src/test/config.sol

```diff
@@ -1789,5 +1789,35 @@ contract Config {
             calc_cut:     0,
             lerp:         false
         });
+        afterSpell.collaterals["RETH-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      5 * MILLION,
+            aL_gap:       3 * MILLION,
+            aL_ttl:       8 hours,
+            line:         0,
+            dust:         15 * THOUSAND,
+            pct:          150,
+            mat:          17000,
+            liqType:      "",
+            liqOn:        false,
```

Comment:
Is this correct? It should probably be:
```suggestion
            liqType:      "clip",
            liqOn:        true,
```

Reply:
Yes this was a residual from predeployment of the Oracle - fixed.